### PR TITLE
return null instead of json_encode -- fixes issue with other column t…

### PR DESCRIPTION
**Pull request by ryanm352**
…ypes such as varchar, text for mysql < 5.7

### CODE REVIEW

File: resources/views/master.blade.php [hunk 1]
```diff
@@ -43,6 +43,7 @@
     @if(!empty(config('voyager.additional_css')))<!-- Additional CSS -->
         @foreach(config('voyager.additional_css') as $css)<link rel="stylesheet" type="text/css" href="{{ asset($css) }}">@endforeach
     @endif
+	@stack('css')
```
Comment (fletch3555) [inline resources/views/master.blade.php:46]: This change is unrelated to the PullRequest title.  Please remove it

**Comment by DaLooK** [comment]
Issue already fixed by pull request #4158
